code/src/index.css

```diff
@@ -1,13 +1,191 @@
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  margin: 20px;
+  font-family: "Roboto Mono", monospace;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #f5f5f1;
+  background-image: linear-gradient(#f5f5f1 0%, #e50914 54%);
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+textarea {
+  resize: none;
+  width: 90%;
+  border: 2px solid rgb(150, 149, 149);
+  overflow-x: hidden;
+}
+
+.new-thought-input {
+  margin: 0 auto;
+  display: flex;
+}
+
+.red-text {
+  color: red;
+}
+
+h1 {
+  text-align: center;
+  color: pink;
+  text-shadow: 2px 2px red, 3px 3px red, 4px 4px red;
+
```

Comment:
One line of space that is not needed. (WHY even bother to say that haha, but I guess it is good to get a reminder on looking over the code twice 😅 and not much else than that to give useful feedback on...)